methods.py

```diff
@@ -1048,6 +1048,40 @@ def progress_finish(target, source, env):
     AlwaysBuild(progress_finish_command)
 
 
+def validate_target_filename(suffix):  # Check if target file can be written to.
+    if os.access("bin/godot" + suffix, os.F_OK) and not os.access("bin/godot" + suffix, os.W_OK):
+        suffixes = suffix.split(".")  # Split suffix string so we can add suffix number later
+        extension = suffixes.pop()  # Store last suffix, should be the file extension.
+
+        old_suffix = suffix  # Store old suffix to use in print statement
+        suffix = ""
+
+        MAXIMUM_ATTEMPTS = 10
+        for suffix_number in range(1, MAXIMUM_ATTEMPTS + 1):
+            new_suffix = ".".join(suffixes) + "." + str(suffix_number) + "." + extension
```

Comment:
```suggestion
            new_suffix = f"{name}.{suffix_number}.{extension}"
```
followup for the suggestion above

Reply:
Thanks!